src/main/webapp/CreateProfilePage.html

```diff
@@ -9,25 +9,32 @@
   <body>
     <div class="container">
       <h1 class="text-center">Create Your Profile Page</h1>
-      <form action="/profile-data" method="POST">
+      <form action="/profile-data" method="POST" novalidate>
         <div class="form-group col-md-5" style="margin: 0 auto;">
           <label for="name">Name:</label><br />
           <input
             type="text"
             class="form-control"
             name="name"
             maxlength="160"
+            required
```

Comment:
Does this prevent the user from submitting all spaces? something like "     "?

Reply:
I just tested it now. It seems to let the users input a " "

Reply:
You can also add a regex pattern for Name and Bio, if you want to prevent users from submitting spaces or random characters.